.github/workflows/test-environment.yml

```diff
@@ -0,0 +1,102 @@
+name: sanity workflow
+
+on:
+  # Ability to execute on demand
+  workflow_dispatch:
+    inputs:
+      elk-stack-version:
+        required: true
+        description: "Elastic Cloud stack version"
+        default: "8.8.0-SNAPSHOT"
+      ess-region:
+        required: true
+        description: "Elastic Cloud deployment region"
+        default: "gcp-us-west2"
+
+env:
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+  AWS_REGION: "eu-west-3"
+  TF_VAR_ec_api_key: ${{ secrets.CI_EC_API_KEY }}
+  EC_DEPLOYMENT: "module.ec_deployment"
+  WORKING_DIR: deploy/cloud
+
+jobs:
+  Setup:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 15
+    outputs:
+      stack-version: ${{ steps.set-elk-vars.outputs.stack-version }}
+      ess-region: ${{ steps.set-elk-vars.outputs.ess-region }}
+      tf-state: ${{ steps.gen-tf-name.outputs.tf-file}}
+    steps:
+      - id: set-elk-vars
+        if: github.event_name == 'workflow_dispatch'
```

Comment:
Why do we need that if anyway that's the only trigger?

Reply:
You are right, for now it can be removed.